api-reference/10 UI Components/dxForm/1 Configuration/onSmartPasted.md

```diff
@@ -5,19 +5,107 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed after an AI service populates the Form.
 
 ##### param(e): ui/form:SmartPastedEvent
-<!-- Description goes here -->
+Information about the event that caused the function's execution.
 
 ##### field(e.aiResult): Object
-<!-- Description goes here -->
+Smart Paste results. This data populates the Form fields.
 
 ##### field(e.component): {WidgetName}
-<!-- Description goes here -->
+UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
+
+---
+The following code snippet displays a notification that indicates whether Smart Paste is successful:
```

Comment:
"displays a notification that indicates" - all these words are pretty similar. You can display, notify, indicate... So I want to get rid of at least one verb there. 

```suggestion
The following code snippet displays a notification with Smart Paste operation result:
```